pkg/tfbridge/info.go

```diff
@@ -155,11 +156,13 @@ type ResourceInfo struct {
 	Aliases             []AliasInfo            // aliases for this resources, if any.
 	DeprecationMessage  string                 // message to use in deprecation warning
 	CSharpName          string                 // .NET-specific name
+	ImportDetails       string                 // Docs overwrite for import instructions
```

Comment:
This should be a proper Golang doc string. Could also change the other properties in the struct.

Reply:
Did you have a certain format in mind?